packages/fleet/charts/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: fleet
+version: v0.2.0
+appVersion: v0.2.0
+icon: https://raw.githubusercontent.com/StrongMonkey/fleet/master/fleet.svg
```

Comment:
This should be hosted in Rancher somewhere.

Reply:
This is currently a hack. I will try to get that svg file into rancher/fleet and then use it from there.